packages/asc-ui/src/utils/themeUtils.ts

```diff
@@ -41,23 +41,32 @@ export const breakpoint = (
   return breakpointFunc && breakpointFunc(type)
 }
 
-const generateCSSFromTypography = ({
-  defaultColor,
-  fontWeight,
-  fontSize,
-  letterSpacing,
-  lineHeight,
-  marginBottom,
-}: any) => css`
+const generateCSSFromTypography = (
+  {
+    defaultColor,
+    fontWeight,
+    fontSize,
+    letterSpacing,
+    lineHeight,
+    marginBottom,
+  }: any,
+  skipAutoMargin?: boolean,
```

Comment:
this naming is not the best choice because is specific to margin-bottom. Maybe something like noMarginBottom? The construction of the function is very specific, don't have any sugesstiion now but please review this, maybe it can be done in a different way

Reply:
good one! I will instead pass the existing gutterBottom instead, since we already have it. we can later rename this one as well, but lets make sure it's backwards compatible